haystack/modeling/model/adaptive_model.py

```diff
@@ -356,7 +356,7 @@ def prepare_labels(self, **kwargs):
             all_labels.append(labels)
         return all_labels
 
-    def forward(self, **kwargs):
+    def forward(self, output_hidden_states: bool = False, output_attentions: bool = False, **kwargs):
```

Comment:
Please add the doc strings for the new parameters here as well, e.g.:
```
 :param output_hidden_states: Whether to output hidden states
 :param output_attentions: Whether to output attentions
```

Reply:
I have added the doc strings.